spec/server_spec.lua

```diff
@@ -84,4 +107,38 @@ describe("http.server module", function()
 		assert.truthy(cq:empty())
 		assert.spy(on_stream).was.called()
 	end)
+	--[[
+	--
+	-- Until there is a way to generate OpenSSL contexts in this file for
+	-- UNIX domain sockets, there is no way to use TLS with this. Because
+	-- of this, the status for using TLS with UNIX domain sockets is
+	-- pending.
+	--
+	--]]
+	local socket_path = os.tmpname()
```

Comment:
Is there a reason to? If you remove it in the finally() call, it should be done after the it() call, right?